README.md

```diff
@@ -442,3 +442,91 @@ receiver.addHandler(new ExampleReceiver() {
 });
 connection.call("interfaceName", "methodName", args, receiver);
 ```
+
+#### Proxy
+
+`Proxy` is a class generated to provide user-friendly interface to your
+`Connection`. To create proxy, you just need to describe the wished interface
+with annotations.
+
+To declare the proxy interface, you need to add `@Proxy` annotation to interface
+definition. If you want your proxy to be a singleton, set the `singleton`
+parameter to `true`. If you want to set the default interface name for the
+Connection methods, you can set it as an `interfaceName` parameter.
+
+```java
+// Creates singleton proxy with default interface name defaultInterface
+@Proxy(interfaceName = "defaultInterface", singleton = true)
+public interface MyProxy {
+  // proxy methods
+}
+```
+
+For now, you can declare proxy wrappers for sending `call` and `event` packets.
+Proxy will be extended for other packets handling.
```

Comment:
Not sure about this line, there may be new features but necessarily connected to that.
Anyway s/for other packets handling/to handle other packets.